contributing-guides/git-terminal.md

```diff
@@ -34,7 +34,10 @@ Typically a pull request will include changes in a single file.
 
 # Updating your fork
 
-Forks aren't updated automatically and in order to avoid merge conflicts it should be done manually and regularly. This is possible via either the GitHub web interface by pressing `Fetch upstream` and `Fetch and merge` on the fork or using Git in the terminal:
+Forks aren't updated automatically and in order to avoid merge conflicts it should be done manually and regularly. This is possible via the GitHub web interface by pressing `Fetch upstream` and `Fetch and merge` on the fork:
+![](https://user-images.githubusercontent.com/50295997/119611842-802ff680-bdfb-11eb-8033-1c61df777c30.png).
```

Comment:
Please be a nice a11y guy and add the `alt` text to the image :grin: 

```suggestion

![Fetch and merge button in GitHub](https://user-images.githubusercontent.com/50295997/119611842-802ff680-bdfb-11eb-8033-1c61df777c30.png).
```

Reply:
It might be nice to use a light theme in this image, because that's GitHub's _default_ color theme. 🤔

Reply:
@bl-ue you're right, @marchersimon my suggestion has a screenshot with the default theme.

Reply:
I used my own screenshot, since I didn't really want to have my name eternalized 😄